manifest.yml

```diff
@@ -18,6 +18,8 @@ packages:
         annotations:
           # this is important security wise
           final: true
+       limits:
+          memory: $MEMORY_TVM_ACTION
```

Comment:
Why do we go to such great lengths to make this consistent across all actions deployed for the tvm?  To the point of even storing it in vault ...
The values for `runtime: 'nodejs:22'` are just as likely to change, and just as much a secret, but we don't treat them the same ...
I would prefer to see manifest.yml be the place where you configure the actions and not push complexity into other areas without reason.

Reply:
I disagree. This is about service deployment. We can't do dynamic changes to adjust to changing conditions without changing this in this repo, releasing, then redeploying choosing the right version of aio-tvm. Versus just changing it in the service deploy in Jenkins.

Reply:
resolved via https://github.com/adobe/aio-tvm/pull/191/commits/6b6eabb8bffc1ad990b9eca622d6357181c44fc7